web_company_color/view/res_company.xml

```diff
@@ -28,6 +28,7 @@
                         <field name="color_button_bg" widget="color" />
                         <field name="color_button_bg_hover" widget="color" />
                         <field name="color_button_text" widget="color" />
+                        <field name="color_input_bg" widget="color" />
```

Comment:
Since it only affects required fields, a more specific name would be better.
```suggestion
                        <field name="color_required_input_bg" widget="color" />
```